### DISCUSSION TITLE
High RAM usage

### DISCUSSION THREAD

**Discussion by sarihindiba** [sunduanchen/Scissor#64] [Q&A]
Hi, I am aiming to run Scissor on a single-cell dataset containing nearly 300,000 cells. Each time I try to run it, it uses over 1500 GiB of RAM, reaching my server's capacity and causing me to interrupt the process. I am new to bioinformatics, so I was wondering if you have any recommendations to reduce memory usage for my dataset. I tried downsampling my data but encountered the same issue even with 100,000 cells.

**Comment by lxpsxx** [comment]
https://github.com/lxpsxx/LargeScissor
